logging-clo-gui-install.adoc

```diff
@@ -0,0 +1,54 @@
+// Module is included in the following assemblies:
+//
+:_mod-docs-content-type: PROCEDURE
+[id="logging-clo-gui-install_{context}"]
+= Installing the {clo} using the {product-title} web console
+To install and configure logging on your {product-title} cluster, additional Operators must be installed. This can be done from the Operator Hub within the web console.
+
+{Product-title} Operators use custom resources (CR) to manage applications and their components. High-level configuration and settings are provided by the user within a CR. The Operator translates high-level directives into low-level actions, based on best practices embedded within the Operator’s logic. A custom resource definition (CRD) defines a CR and lists all the configurations available to users of the Operator. Installing an Operator creates the CRDs, which are then used to generate CRs.
+
+.Prerequisites
+
+* Supported object store (AWS S3, Google Cloud Storage, Azure, Swift, Minio, OpenShift Data Foundation)
+
+.Procedure
+
+. To install the {clo} in the {product-title} web console, click *Operators* -> *OperatorHub*.
+
+.. Type `OpenShift Logging` in the filter by keyword box. Choose *Red Hat OpenShift Logging* from the list of available Operators and click *Install*.
+
+.. On the Install Operator page, under *Update channel* select *stable*.
```

Comment:
```suggestion
.. On the *Install Operator* page, under *Update channel* select *stable*.
```

Reply:
I think this needs to be bold since its the name of a UI page. I see it bold in other place of the repo.